app/Http/Controllers/AccountController.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Account;
+use function CsrDelft\getDateTime;
+use CsrDelft\model\entity\security\AccessRole;
+use function CsrDelft\setMelding;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Facades\Validator;
+use Illuminate\Validation\Rule;
+
+class AccountController extends Controller
+{
+    const SESSION_SU_KEY = 'suedFrom';
+
+    public function __construct()
+    {
+        parent::__construct();
+        $this->middleware('auth');
+    }
+
+    /**
+     * @param Account|null $account
+     * @return \Illuminate\Contracts\View\Factory|\Illuminate\View\View
+     * @throws \Illuminate\Auth\Access\AuthorizationException
+     */
+    public function bewerken(Account $account)
+    {
+        $this->authorize('view', $account);
+
+        return view('account.bewerken', ['account' => $account]);
```

Comment:
Eerlijk gezegd vind ik deze manier van het bouwen van een formulier minder mooi dan de methode die we hadden (mbv de Formulier class). Misschien kunnen we iets als https://github.com/kristijanhusak/laravel-form-builder gebruiken, hoewel daar ook wel een aantal nadelen aan kleven.

Reply:
Eens, ik denk dat het het beste is om de huidige formulier klasse te blijven gebruiken, werkt prima met Laravel samen en er zit best wel wat superspecifieke functionaliteit in. Verder vind ik formulier zoals je nu is best wel nice werken (alleen even een keer die Java script los trekken)

Reply:
Ja, de Formulier klasse werkt best goed. Daar zitten natuurlijk ook wel een paar verbeterpuntjes aan, maar daar kunnen we eventueel iets aan doen.